alarms.json.default

```diff
@@ -38,7 +38,16 @@
 			"access_secret": "YOUR_ACCESS_SECRET",
 			"consumer_key": "YOUR_CONSUMER_KEY",
 			"consumer_secret": "YOUR_CONSUMER_SECRET"
-        }
+        },
+		{
+			"active": "False",
+			"type":"ifttt",
+			"api_key":"bryeq0dWmIgBlDxUiBWaQ3",
```

Comment:
Aaaaand I knew I'd leave this in by mistake. I'll revoke it from my account, I guess you can just fix this before merging?